src/main/comments-store.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+
+class CommentsStore {
+  constructor() {
+    this.map = new WeakMap();
```

Comment:
This class have `get` method so can you make `this.map` private?

Reply:
Private properties not supported in Node.js versions we support. And it's used https://github.com/prettier/prettier/pull/9830/files#diff-129b21c2ee63fc0f00abf2c578cb05d619f4097c26823f2bf21e335b46ad52e8R1534

Reply:
I know we cannot use private field. I mean soft private by TypeScript compiler with JSDoc comments:

```js
/**
@private
*/
```

But I got that `this.map` is used.